packages/agoric-cli/src/init.js

```diff
@@ -2,8 +2,10 @@ import chalk from 'chalk';
 import { makePspawn } from './helpers.js';
 
 // Ambient types. Needed only for dev but this does a runtime import.
+// https://github.com/Agoric/agoric-sdk/issues/6512
 import '@endo/captp/src/types.js';
 import '@agoric/swingset-vat/exported.js';
+import '@agoric/swingset-vat/src/vats/network/types.js';
```

Comment:
Seeing this now, would it make sense to add `import './src/vats/network/types.js';` to `@agoric/swingset-vat/exported.js` instead ?

Edit: maybe not since it's used directly in other places?

Reply:
yeah, I considered that and opted for lesser change to what's exported. I figure it will be addressed in further ambients work